docs/components/InputPassword/Mobile.stories.tsx

```diff
@@ -7,6 +7,8 @@ export default {
   component: InputPassword,
   parameters: {
     viewMode: "story",
+    previewTabs: { code: { hidden: false } },
```

Comment:
Is our default for this setting `true` or do we have no default, i.e. always need to specify it?

Reply:
@JeffMarvin  Yup! In `../.storybook/preview.tsx` it's marked `true` by default.  

My understanding is when we implemented the code tab, we wanted to hide it by default because the mobile stories didn't have the code tab fixed yet.  If we took this away now, or changed the default to false, the code tab would appear everywhere which I don't think we are ready for yet.  
It will show up on the Docs page and then show the code for the first example.
And because not everything is finalized yet, shipping a change now might not be best/add confusion to the user.  (I know you aren't asking that yet but wanted to share some considerations)

Reply:
Sounds good, thanks for the additional context! :)